tests/template_tests/test_response.py

```diff
@@ -221,6 +221,12 @@ def test_pickling_cookie(self):
 
         self.assertEqual(unpickled_response.cookies['key'].value, 'value')
 
+    @override_settings(ROOT_URLCONF='template_tests.urls',
```

Comment:
Haha, I was thinking about putting it in the handlers tests and @freakboy3742 told me to put that here. ;)